scripts/basic-workflow-tests.sh

```diff
@@ -163,6 +163,14 @@ test_convert() {
     ilab model convert
 }
 
+test_evaluate() {
+    task Evaluating a model
+    INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true 
+    HF_DATASETS_TRUST_REMOTE_CODE=true 
+    ilab model download --repository instructlab/granite-7b-lab
```

Comment:
Should we add that step in the `test_download` function?

Reply:
yeah, makes sense to me

Reply:
Actually, wouldn't the user workflow be to run this against the output of `ilab model train` ?

Reply:
In theory, this can be used to evaluate any model (checkpoint or otherwise) - so it boils down to what we want to test here.